bower.json

```diff
@@ -9,14 +9,15 @@
   },
   "main": "dist/fxpay.min.js",
   "ignore": [
-    ".bowerrc",
-    ".gitignore",
-    ".jshintrc",
-    ".travis.yml",
+    "**/.*",
+    "Gruntfile.js",
     "build",
     "example",
-    "node_modules",
     "karma.conf.js",
+    "lib",
```

Comment:
I was thinking including lib would be nice when we support RequireJS (https://bugzilla.mozilla.org/show_bug.cgi?id=1121746). Or do you think it would be better for projects (like Fireplace) to work from the repo source when using RequireJS for local debugging?